package.json

```diff
@@ -27,6 +27,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "tabbable": "^4.0.0",
```

Comment:
This dependency was chosen intentionally to be deduplicated, since it's used internally by another of our dependencies: `focus-trap` ([source](https://github.com/davidtheclark/focus-trap/blob/2a03f59996f2d60c98d8e437ca817f71e4815ec9/package.json#L39)). It occurs to me now that, since we're using an out-of-date version of `focus-trap` (2.3.0, vs. current 5.1.0), that this will not be deduplicated, and we'll be loading both versions.

A separate effort should seek to update the version of `focus-trap` so that we can take advantage of the deduplication (avoid shipping two copies of the library to users).